app/i18n/app.fr.i18n.json

```diff
@@ -1,7 +1,7 @@
 {
     "app": {
         "loading": "Chargement...",
-        "offline": "Impossible de se connecter, êtes-vous hors-ligne?",
+        "offline": "Impossible de se connecter, êtes-vous hors-ligne ?",
```

Comment:
Personally I hate spaces before punctuation, but I do understand this is default in french typesetting..